### PR TITLE
BigQuery Stream documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added documentation, images, and revised sidebar structure for a new section called Streaming platform and the BigQuery Stream destination.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by sanpj2292** [comment]
The PR looks good. I will approve it.

@ameypv-rudder Don't merge this yet as the release is planned for 29th Dec, 2021. Let's merge it then

**Review by sanpj2292 — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
> The PR looks good. I will approve it.
> 
> @ameypv-rudder Don't merge this yet as the release is planned for 29th Dec, 2021. Let's merge it then

Sure - added a do not merge label. Thanks for the approval!

File: destinations/streaming-platforms/bigquery-stream.mdx [hunk 1]
```diff
@@ -0,0 +1,185 @@
+---
+# slug: "/docs/destinations/streaming-platforms/bigquery-stream"
+title: "BigQuery Stream"
+description: Step-by-step guide to stream your event data from RudderStack to Google BigQuery.
```
Comment (sanpj2292) [inline destinations/streaming-platforms/bigquery-stream.mdx:4]:
```suggestion
description: Step-by-step guide to stream your event data from RudderStack to Google BigQuery via streaming api.
```
Probably this looks more appropriate

File: destinations/streaming-platforms/bigquery-stream.mdx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            [a].concat(Array.prototype.slice.call(arguments))
+          );
+        };
+      })(method);
+    }
+    rudderanalytics.load("<WRITE_KEY>", "<DATA_PLANE_URL>");
+
+    rudderanalytics.track("relevant event name", {
+      productId: 10,
+      productName: `Product-10(From JSSDK - to DataPlane)`,
```
Comment (sanpj2292) [inline destinations/streaming-platforms/bigquery-stream.mdx:164]:
```suggestion
      productName: `Product-10`,
```
Can we change in this way ?

**Pull request closed without merging**